cmd/create/machinepool/nodepool.go

```diff
@@ -372,7 +378,13 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		npBuilder.TuningConfigs(inputTuningConfig...)
 	}
 
-	npBuilder.AWSNodePool(cmv1.NewAWSNodePool().InstanceType(instanceType))
+	awsNpBuilder := cmv1.NewAWSNodePool().InstanceType(instanceType)
+
+	if len(securityGroupIds) > 0 {
+		awsNpBuilder.AdditionalSecurityGroupIds(securityGroupIds...)
+	}
+
+	npBuilder.AWSNodePool(awsNpBuilder)
```

Comment:
NP : move this to its own func, just to write a simple test to ensure we catch security group ids